modules/reference/pages/rpk/rpk-cluster/rpk-cluster-config-status.adoc

```diff
@@ -13,6 +14,18 @@ it is incremented each time a configuration change is applied to the
 cluster. If a broker is using an earlier version as indicated by a lower number,
 it may be out of sync with the rest of the cluster. This can happen if a broker
 is offline or if it has not yet applied the latest configuration changes.  The cluster configuration version number is not the same as the Redpanda version number.
+endif::[]
+
+ifdef::env-cloud[]
+Check the progress of a cluster configuration change.
+
+Some cluster properties require a rolling restart when updated, and it can take several minutes for the update to complete. This command lists the long-running operations executed by the update and their status:
+
+- In progress
+- Completed
+- Failed
+
+endif::[]
```

Comment:
It'd be nice to show an example of the output

Reply:
@andresaristizabal would you mind providing that here? Thanks!

Reply:
Sure:

example about set:
```
rpk cluster config set iceberg_enabled=true iceberg_catalog_type=object_storage
Processing configuration, this operation may take up to 10 minutes. To check the status, run 'rpk cluster config status'

Operation ID: d0ec0sor49uba166af3g
```

example about status:

```
 rpk cluster config status
OPERATION-ID          STATUS   STARTED              COMPLETED
d0ec1obmpnr7lv17bfpg  RUNNING  2025-05-08 14:34:09
d0ec0sor49uba166af3g  RUNNING  2025-05-08 14:32:20
```